src/scene/data.ts

```diff
@@ -122,57 +75,34 @@ export async function loadFromJSON() {
     return { elements, appState };
   };
 
-  if ("chooseFileSystemEntries" in window) {
-    try {
-      (window as any).handle = await (window as any).chooseFileSystemEntries({
-        accepts: [
-          {
-            description: "Excalidraw files",
-            extensions: ["json"],
-            mimeTypes: ["application/json"]
+  const blob = await fileOpen({
+    description: "Excalidraw files",
+    extensions: ["json"],
+    mimeTypes: ["application/json"]
+  });
+  if (blob.handle) {
+    (window as any).handle = blob.handle;
+  }
+  let contents;
+  if ("text" in Blob) {
+    contents = await blob.text();
+  } else {
+    contents = await (async () => {
+      return new Promise(resolve => {
+        const reader = new FileReader();
+        reader.readAsText(blob, "utf8");
+        reader.onloadend = () => {
+          if (reader.readyState === FileReader.DONE) {
+            resolve(reader.result as string);
           }
-        ]
-      });
-      const file = await (window as any).handle.getFile();
-      const contents = await file.text();
-      const { elements, appState } = updateAppState(contents);
-      return new Promise<DataState>(resolve => {
-        resolve(restore(elements, appState));
+        };
       });
-    } catch (err) {
-      if (err.name !== "AbortError") {
-        console.error(err.name, err.message);
-      }
-      throw err;
-    }
-  } else {
-    const input = document.createElement("input");
-    const reader = new FileReader();
-    input.type = "file";
-    input.accept = ".json";
-
-    input.onchange = () => {
-      if (!input.files!.length) {
-        alert("A file was not selected.");
```

Comment:
We've lost this one alongside this PR. (Also reading the package source, it seems you're not doing this check at all. Is that intentional?).

Reply:
It's now happening in https://github.com/tomayac/browser-nativefs/blob/master/src/file-open-legacy.mjs (and silently ignored).